Add Hugo theme (#113)
pyladies/global-conference: content/_index.md

+++
title = 'Home'
draft = false
layout = 2
+++

{{< rawhtml >}}
<!-- TODO: Update banner -->
<div style="background: #D2E0FB;" class="position-relative overflow-hidden p-1 p-md-3 m-md-1 text-center">
    <div class="d-md-flex flex-md-equal w-75 my-md-3 mx-auto align-items-center">
      <div class="col-md-5 p-lg-3 mx-auto my-5">

>         <blockquote class="blockquote text-center">
>           <p>
>             <i class="fas fa-quote-left fa-xs pink" aria-hidden="true"></i>
>             <span>The more I study, the more insatiable do I feel my genius for it to be.</span>
>             <i class="fas fa-quote-right fa-xs pink" aria-hidden="true"></i>
>           </p>
>           <footer class="blockquote-footer">Ada Lovelace</footer>
>         </blockquote>

Author: ```suggestion
```

        <h1 class="pink display-4 fw-normal">PyLadiesCon</h1>
        <h4 class="gradient special-font">2024 Online</h4>
        <p class="lead fw-normal">

>           TBD December | Multi-language | Multi-timezone

Author: ```suggestion
          6th-8th December | Multi-language | Multi-timezone
```

        </p>
        <div class="social w-50 mx-auto pt-3">
          <a href="https://fosstodon.org/@pyladiescon"><i class="fab fa-mastodon fa-2x px-2" aria-hidden="true"></i></a>
          <a href="https://twitter.com/pyladiescon"><i class="fab fa-twitter fa-2x px-2" aria-hidden="true"></i></a>
          <a href="https://instagram.com/pyladiescon"><i class="fab fa-instagram fa-2x px-2" aria-hidden="true"></i></a>
          <a href="https://www.linkedin.com/company/pyladiescon"><i class="fab fa-linkedin fa-2x px-2" aria-hidden="true"></i></a>
        </div>
      </div>
      <div class="col-md-5 p-lg-1 mx-auto my-5">
        <img style="width: 60%;" src="img/banner.png">
      </div>
  </div>
</div>
{{< /rawhtml >}}

{{% mainrectangle %}}
![About Icon class=icon-section](/img/about-icon.png)

{{< pinktitle "About the conference" >}}

Attention all PyLadies community members! We’re excited to share that we are in
the early stages of planning a PyLadies Conference (PyLadiesCon),
a transformative event designed to promote diversity, learning, and empowerment
within the Python community.  🎉

> Save the date! The conference will take place on the first weekend of December,

Author: ```suggestion
Save the date! The conference will take place on December 6th-8th,
```
